internal/pkg/cli/cli.go

```diff
@@ -180,3 +180,11 @@ func displayPath(target string) string {
 	}
 	return rel
 }
+
+func isDir(fs afero.Fs, path string) (bool, error) {
```

Comment:
<img src="https://emojis.slackmojis.com/emojis/images/1643514599/6023/meow_peek.png" width="20" height="20" /> does this actually shorten any logic?🥺